lib/ProductOpener/Images.pm

```diff
@@ -1253,24 +1253,11 @@ HTML
 		}
 	}
 
-	# If we don't have an image, display Pacman
+	# No image
 	if ($html eq '') {
 
-		my @colors = qw(
-ff6600
-ffcc00
-55d400
-00ccff
-0066ff
-ff00cc
-cc00ff
-);
-		my $color_id = $product_ref->{code} % (scalar @colors);
-		my $color = $colors[$color_id];
-
 		$html = <<HTML
-<div style="background-color:#$color">
-<img src="$static/images/misc/pacman.svg" width="$thumb_size" height="$thumb_size" alt="Please add pictures of the product if you have it!" $css/>
+<div style="background-color:#eeeeee" style="width:$thumb_size;height:$thumb_size">
```

Comment:
Just an empty div? Wouldn't it be better with a placeholder?

Reply:
Well the empty div is simple and does the job, what kind of placeholder do you have in mind?

Reply:
I was thinking of something like a question mark.